src/integration/pages/frontPage/crossPlatformTests.js

```diff
@@ -8,4 +8,53 @@ export default () => {
 
     expect(sect).toBeInTheDocument();
   });
+
+  it('I can see Index Alsos', () => {
+    const topStories = document.querySelector(
+      '[aria-labelledby="Top-stories"]',
+    );
+
+    if (topStories) {
+      const indexAlsos = topStories.querySelector('[data-e2e=index-alsos]');
+
+      if (indexAlsos) {
+        const h4 = indexAlsos.querySelector('h4');
+        expect(h4.textContent).toMatchSnapshot();
+
+        const data = window.SIMORGH_DATA;
+
+        if (data) {
+          const topStoriesGroup = data.pageData.content.groups[0].items[0];
+          const { relatedItems } = topStoriesGroup;
+
+          if (relatedItems.length > 1) {
+            expect(indexAlsos.getElementsByTagName('ul')).toHaveLength(1);
+            expect(indexAlsos.getElementsByTagName('li')).toHaveLength(
+              relatedItems.length,
+            );
+          } else {
+            const { headline } = relatedItems[0].headlines;
+
+            expect(indexAlsos.querySelector('div a span').innerHTML).toEqual(
```

Comment:
Is a snapshot maybe better here? At least that way we can see the text which also enables us to sanity check the outputted text